src/styles/blocks/calendar.scss

```
@@ -0,0 +1,60 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+
+  max-width: $day-size * $columns + $day-gap * ($columns - 1);
+
+  $week-days:
+    "mon" 1,
+    "tue" 2,
+    "wed" 3,
+    "thu" 4,
+    "fri" 5,
+    "sat" 6,
+    "sun" 7;
+
+  @each $day, $order in $week-days {
+    &--start-day--#{$day} {
+      & > :first-child {
```

```suggestion
      & :first-child {
```
Без этого тоже должно работать